File: 1. 단계별로_풀어보기/1~9/1. 입출력과 사칙연산/10. 2588.js
const fs = require('fs')
const inputs = fs.readFileSync('/dev/stdin').toString().split('\n')

let [num1, num2] = inputs

num1 = parseInt(num1)
num2 = parseInt(num2)

const num2_1 = num2 % 10
const num2_10 = Math.floor(num2 / 10) % 10
const num2_100 = Math.floor(num2 / 100)

// prettier-ignore
const resultArr = [
  num1 * num2_1,
  num1 * num2_10,
  num1 * num2_100,
  num1 * num2
]

process.stdout.write(resultArr.join('\n'))

File: 1. 단계별로_풀어보기/1~9/1. 입출력과 사칙연산/10. 2588.py
import sys
import math

input = sys.stdin.readline
output = sys.stdout.write

num1 = int(input())
num2 = int(input())

num2_1 = num2 % 10
num2_10 = (num2 // 10) % 10
num2_100 = num2 // 100

output(f"{num1 * num2_1}\n")
output(f"{num1 * num2_10}\n")
output(f"{num1 * num2_100}\n")
output(f"{num1 * num2}\n")